src/main/java/mate/controller/AddCarController.java

```diff
@@ -0,0 +1,42 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Manufacturer> allManufacturersList = manufacturerService.getAll();
+        req.setAttribute("maxId", allManufacturersList.stream()
+                .mapToLong(Manufacturer::getId)
+                .max()
+                .orElse(1L));
```

Comment:
It's really bad practice to get all records from db and then filter them. If you want to get the maximum index from a table in the database, you need to create a separate query to the database. But initially I do not see how this information can be used, you may not have records with a specific index in the database so the maximum index is not particularly informative